CMakeLists.txt

```diff
@@ -1627,7 +1662,7 @@ if (LWS_WITH_GENERIC_SESSIONS)
 		      plugins/generic-sessions/assets/index.html
 		DESTINATION share/libwebsockets-test-server/generic-sessions
 			COMPONENT examples)
-	install(FILES plugins/generic-sessions/assets/successful-login.html 
+	install(FILES plugins/generic-sessions/assets/successful-login.html
```

Comment:
Although I don't like the trailing spaces, we should not touch lines unnecessarily when preparing a patch for upstream.  Revert whitespace changes for code that would not otherwise be modified.